app/apollo/subscription/index.js

```diff
@@ -99,7 +102,31 @@ class PubSubImpl {
     }
     return false;
   }
-
+  async livenessCheckSubscriber(pubsubQueue){
+    if (process.env.NODE_ENV !== 'unit-test' && process.env.NODE_ENV !== 'test') {
+      const sizeLimit = 10;
+      this.pubSub.redisSubscriber.on('message', function(channel, message){
+        //console.log('receiving: ', JSON.parse(message));
```

Comment:
we need to filter channel here since we only want to store the test data for channel == 'testTopic'